dissect/target/plugins/apps/vpn/openvpn.py

```diff
@@ -166,14 +166,18 @@ def _parse_config(content: str) -> dict[str, Union[str, list[str]]]:
     """Parses Openvpn config  files"""
     lines = content.splitlines()
     res = {}
+    boolean_fields = OpenVPNServer.getfields("boolean") + OpenVPNClient.getfields("boolean")
 
     for line in lines:
         # As per man (8) openvpn, lines starting with ; or # are comments
         if line and not line.startswith((";", "#")):
             key, *value = line.split(" ", 1)
-            value = value[0] if value else ""
-            # This removes all text after the first comment
-            value = CONFIG_COMMENT_SPLIT_REGEX.split(value, 1)[0].strip()
+            if any(key == boolean_field.name for boolean_field in boolean_fields):
+                value = value[0] if value else True
+            else:
+                value = value[0] if value else ""
+                # This removes all text after the first comment
+                value = CONFIG_COMMENT_SPLIT_REGEX.split(value, 1)[0].strip()
```

Comment:
I think we also need to deal with comments in configuration lines with boolean fields and split them off.

I was thinking along the lines:
```
key, _, value = line.partition(" ")
# This removes comments and whitespace around the value
value = CONFIG_COMMENT_SPLIT_REGEX.split(value, 1)[0].strip()
if not value and key in boolean_field_names:
    value = True
```